### ISSUE TITLE
PyTorch maDLC on 1 indiv creates saDLC tracks which fail in create_labeled_video

### ISSUE THREAD

**Issue by backyardbiomech** [DeepLabCut/DeepLabCut#2741]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I have a long-running maDLC project which I recently decided to try with PyTorch. While it is maDLC, it always only tracks a single animal.
The new model trained to 120 epochs with `batch_size = 8`, and evaluated to pixel errors about 2.1, which is as good (or better) than the last tensorflow model.
Upon analyzing a new video with `auto_track=True`, I now get three output files:

- `...shuffle1_snapshot_115.h5`
- `...shuffle1_snapshot_115_full.pickle`
- `...shuffle1_snapshot_115_meta.pickle`

So, unlike the previous version, I don't get `assemblies` or `..._el.h5` for the tracker.

The `full.pickle` file appears rather empty when I load it directly, and `create_video_with_all_detections` creates a video with no markers and prints errors for each frame indicating no data exists.
The h5 file appears to be assembled tracks, which is great. However, it doesn't have the `_el` that downstream methods (like `create_labeled_video` are looking for. I can add the _el to the name, but the file is also missing the "indiv" level of the multi_index. 

So short of changing my config to not be maDLC, the downstream methods don't work. This includes `create_labeled_video` and `filterpredictions` at least. Both of which return an error like

```
Filtering with median model C:\Users\...\videos-raw\cam1_28-Mar-24_18-00-34.mov
No unfiltered data file found in C:\Users\...\videos-raw for video cam1_28-Mar-24_18-00-34 and scorer DLC_DlcrnetStride16Ms5_...shuffle1_snapshot_115 and ellipse tracker.
```

By the way, if I edit my config file to pretend to be a single-animal project, filtering and creating videos works without a problem.

### Operating System

Windows 10

### DeepLabCut version

DLC 3.0.0rc4

### DeepLabCut mode

multi animal

### Device type

gpu

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by backyardbiomech** [comment]
I just wanted to add though, that the PyTorch pose estimates with default settings so far are a VAST improvement in accuracy over the previous Tensorflow models, which I had tried all kinds of settings on over the past few years. Thank you DLC team for all of the work on this!

**Comment by n-poulsen** [comment]
hey @backyardbiomech - thanks for reporting this, I'm looking into it. I believe the issue is that we're considering it a single animal project if there's only 1 animal labeled. I think it should be an easy fix.

**Comment by backyardbiomech** [comment]
To add to this issue, it gets worse than just making labeled videos. 
Extracting outlier frames also fails because it can't find the analyzed data. I can trick it into extracting frames by modifying the config to make it a single animal project temporarily. But the machinelabels and collectedData files for the newly extracted videos will be single animal rather than maDLC like all the others in the project. 
I'm assuming I'll have to write something to convert the saDLC collected data files to maDLC before merging and creating the next dataset, which is fine (really just modifying the multi-index header).

**Comment by n-poulsen** [comment]
Thanks for the update - I'm more or less done fixing video analysis, and I'll check that I can extract outlier frames in this scenario as well

**Comment by n-poulsen** [comment]
Fixed!
